robot/robot.py

```diff
@@ -76,4 +91,4 @@ def teleopPeriodic(self):
 
 
 if __name__ == '__main__':
-    wpilib.run(Bot)
+    wpilib.run(Bot, physics_enabled=True)
```

Comment:
It took me a while to figure out why you had this, but then I checked the examples. Funnily enough, this doesn't actually do anything. Even if you set it to false 😂